### ISSUE TITLE
don't work after update

### ISSUE THREAD

**Issue by AntonVLA** [icewind1991/files_markdown#133]

**Comment by AntonVLA** [comment]
help

**Comment by AntonVLA** [comment]
![image](https://user-images.githubusercontent.com/38549992/64976992-33241f00-d8bb-11e9-9588-81bbfc16a289.png)

**Comment by icewind1991** [comment]
Can you see if there is anything in the browsers console log

**Comment by phiL0co** [comment]
I can affirm this!
I'm using NC 15.0.11. After updating to Markdown Editor 2.1.0 the preview does not work.

**Comment by icewind1991** [comment]
You can try updating to nc16 if possible and see if that makes a difference

**Comment by phiL0co** [comment]
Unfortunately I can't because my Ubuntu version doesn't meet the requirements and I don't want to dist-upgrade yet.
It would be great if it would work on NC15, too!

**Comment by jordanbray** [comment]
I can confirm the same problem on NC 16.0.3 with Markdown 2.1.0.

**Comment by jordanbray** [comment]
Since you asked for the console log, I got the following error which looks like the source to me:

```
Unhandled promise rejection TypeError: "n is not a constructor"
    initPromise PreviewPlugin.ts:24
    o main.js:230
    H main.js:230
    u main.js:235
main.js:230:27763
    t https://xxx/core/js/dist/main.js:230
    exports https://xxx/core/js/dist/main.js:235
    O https://xxx/core/js/dist/main.js:230
    exports https://xxx/core/js/dist/main.js:230
    <anonymous> https://xxx/core/js/dist/main.js:181
    v https://xxx/core/js/dist/main.js:181
    y https://xxx/core/js/dist/main.js:181
```

**Comment by jordanbray** [comment]
It magically fixed itself after refreshing a few times.  I was attempting to open a console and see if I could catch the exception, but it vanished as soon as I started looking for it.

**Comment by phiL0co** [comment]
> 
> 
> It magically fixed itself after refreshing a few times. 

What do you mean with that? Is the preview working again for you?

**Comment by jordanbray** [comment]
Yes.  It just started working after a few refreshes.  I didn't change anything server or client side.

**Comment by phiL0co** [comment]
That does not work for me, unfortunately.
My console log gives me:

`TypeError:` r.renderer is undefined                                   PreviewPlugin.ts:71:12`

**Comment by phiL0co** [comment]
Don't ask me why (I didn't update the markdown app), but all of a sudden, the preview is working!

**Comment by datenimperator** [comment]
The preview worked for me, before I upgraded NC 16.0.4 to 16.0.5. After that it did not show the preview anymore (like on OPs screenshot). After clearing the cache and reloading the page, it worked again.

Seems like an issue with stale JS files.
